tests/Factories/RegisterControllerFactoryTest.php

```diff
@@ -25,7 +25,7 @@ function testInvoke()
 
         $container->expects($this->at(1))->method('get')
             ->with($this->equalTo('RandomPasswordModel'))
-            ->willReturn($RandomPasswordModel);
+            ->willReturn('password');
```

Comment:
Which was meant to be used here?

Reply:
The random password model doesn't need to be used, since all it's doing is getting a string out

Reply:
Line 11 is unused

```use Portal\Models\RandomPasswordModel```